src/quest_manager/views.py

```diff
@@ -184,17 +184,25 @@ class QuestCopy(QuestCreate):
     def get_form_kwargs(self):
         kwargs = super().get_form_kwargs()
 
+        quest = get_object_or_404(Quest, pk=self.kwargs['quest_id'])
```

Comment:
why change name from copied_quest to quest?  less info and more ambiguous as to what quest this variable holds.

Reply:
because its not the copied_quest its the original quest. Im not cloning the quest, like in the original code because there is no need to. Im passing the same values from quest to the initial form values

Reply:
Why change it though?

Reply:
I don't mind refactoring, but there's no documentation why this is better?